src/shared/sam/debugger/awsSamDebugConfigurationValidator.ts

```diff
@@ -193,6 +202,24 @@ export class DefaultAwsSamDebugConfigurationValidator implements AwsSamDebugConf
         return { isValid: true }
     }
 
+    private validateApiConfig(debugConfiguration: AwsSamDebuggerConfiguration): ValidationResult {
+        if (!debugConfiguration.api?.path) {
+            return {
+                isValid: false,
+                message: localize(
+                    'AWS.sam.debugger.missingField',
+                    'Missing required field "{0}" in debug config',
+                    'api'
```

Comment:
This isn't quite accurate since this will fail if `api` is present but `api.path` isn't.

Reply:
This technically works as intended (on accident), fixed